packages/web-twig/src/Resources/components/Dropdown/README.md

```diff
@@ -0,0 +1,72 @@
+# Dropdown
+
+This is the Twig implementation of the [Dropdown] component.
+
+Basic example usage:
+
+```twig
+<Button data-toggle="dropdown" data-target="DropdownExample">Open Dropdown</Button>
+<Dropdown id="DropdownExample">Dropdown Content</Dropdown>
+```
+
+Usage with link:
+
+```twig
+<ButtonLink href="javascript:void(0)" data-toggle="dropdown" data-target="CollapseExample">Open Dropdown</ButtonLink>
+...
```

Comment:
Either a comment like `<!-- Dropdown -->` or the actual code of `Dropdown` would be better here, I believe.

Reply:
I have removed this example.